tests/acceptance/Dockerfile.e2e

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.24-alpine AS builder
```

Comment:
Use the ubuntu:24.04 image instead 

Also install the basic tools such as curl, wget nc, sysstat